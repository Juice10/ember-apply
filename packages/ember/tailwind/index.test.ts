import { describe, expect, it } from 'vitest';

import { default as tailwind } from './index';

describe('tailwind', () => {
  it('default export exists', () => {
    expect(typeof tailwind).toEqual('function');
  });
});
