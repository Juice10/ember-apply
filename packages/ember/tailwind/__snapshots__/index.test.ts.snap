// Vitest Snapshot v1

exports[`tailwind > applying to an ember app > works via CLI 1`] = `
" .gitignore       |  3 +++
 app/index.html   |  1 +
 package.json     | 11 ++++++++---
 tests/index.html |  3 ++-
 4 files changed, 14 insertions(+), 4 deletions(-)"
`;

exports[`tailwind > applying to an ember app > works via CLI 2`] = `
"diff --git a/.gitignore b/.gitignore
index 9bab45a..c4e2f3e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,9 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
+
 # compiled output
+public/assets/tailwind.css
+
 /dist/
 /tmp/
 
diff --git a/app/index.html b/app/index.html
index 2398775..4ca37d0 100644
--- a/app/index.html
+++ b/app/index.html
@@ -9,6 +9,7 @@
 
     {{content-for \\"head\\"}}
 
+    <link integrity=\\"\\" rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/tailwind.css\\">
     <link integrity=\\"\\" rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/vendor.css\\">
     <link integrity=\\"\\" rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/test-app.css\\">
 

package.json
{
  \\"name\\": \\"@ember-apply/tailwind\\",
  \\"description\\": \\"ember-apply plugin for tailwind\\",
  \\"version\\": \\"2.0.0\\",
  \\"type\\": \\"module\\",
  \\"license\\": \\"MIT\\",
  \\"preset\\": \\"index.js\\",
  \\"author\\": \\"NullVoxPopuli\\",
  \\"repository\\": {
    \\"url\\": \\"https://github.com/NullVoxPopuli/ember-apply\\",
    \\"type\\": \\"https\\",
    \\"directory\\": \\"packages/ember/tailwind\\"
  },
  \\"keywords\\": [
    \\"ember\\",
    \\"tailwind\\",
    \\"css\\",
    \\"ember-apply\\",
    \\"blueprint\\"
  ],
  \\"exports\\": {
    \\"import\\": \\"./index.js\\"
  },
  \\"files\\": [
    \\"index.js\\",
    \\"files/**/*\\"
  ],
  \\"scripts\\": {
    \\"build:types\\": \\"tsc --build\\",
    \\"lint\\": \\"yarn lint:js\\",
    \\"lint:js\\": \\"eslint .\\",
    \\"lint:fix\\": \\"yarn lint:js --fix\\",
    \\"test\\": \\"vitest --coverage --no-watch\\",
    \\"test:watch\\": \\"vitest --watch\\"
  },
  \\"dependencies\\": [
    \\"ember-apply\\"
  ],
  \\"devDependencies\\": [
    \\"@nullvoxpopuli/eslint-configs\\",
    \\"autoprefixer\\",
    \\"babel-eslint\\",
    \\"c8\\",
    \\"eslint\\",
    \\"eslint-plugin-decorator-position\\",
    \\"eslint-plugin-import\\",
    \\"eslint-plugin-node\\",
    \\"eslint-plugin-prettier\\",
    \\"postcss\\",
    \\"prettier\\",
    \\"tailwindcss\\",
    \\"typescript\\",
    \\"vite\\",
    \\"vitest\\"
  ],
  \\"engines\\": [
    \\"node\\"
  ],
  \\"volta\\": [
    \\"node\\",
    \\"yarn\\"
  ]
}
diff --git a/tests/index.html b/tests/index.html
index ec6650a..a56e778 100644
--- a/tests/index.html
+++ b/tests/index.html
@@ -10,6 +10,7 @@
     {{content-for \\"head\\"}}
     {{content-for \\"test-head\\"}}
 
+    <link rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/tailwind.css\\">
     <link rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/vendor.css\\">
     <link rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/test-app.css\\">
     <link rel=\\"stylesheet\\" href=\\"{{rootURL}}assets/test-support.css\\">
@@ -28,7 +29,7 @@
       </div>
     </div>
 
-    <script src=\\"/testem.js\\" integrity=\\"\\" data-embroider-ignore></script>
+    <script src=\\"/testem.js\\" integrity=\\"\\" data-embroider-ignore=\\"\\"></script>
     <script src=\\"{{rootURL}}assets/vendor.js\\"></script>
     <script src=\\"{{rootURL}}assets/test-support.js\\"></script>
     <script src=\\"{{rootURL}}assets/test-app.js\\"></script>
"
`;
